mission_goals.md

```diff
@@ -0,0 +1,108 @@
+# DID Method Standardization: Mission & Goals
+
+## Introduction
+
+This document outlines the high-level mission, goals, and expected outcomes for the DID Method Standardization initiative. It serves as a strategic guide for stakeholders involved in the development and adoption of [Decentralized Identifiers (DIDs)](https://www.w3.org/TR/did-core).
+
+## Mission
+
+We aim to drive Decentralized Identifier adoption through standardization maturity and ecosystem engagement, as follows:
+
+1. Leading open, collaborative standardization efforts
+2. Identifying approaches that empower organizations to independently drive DID method standardization
+3. Cultivating ecosystem-wide awareness and participation.
+
+## Goals
+
+1. Select, and advance standards for, an initial set of strategically valuable DID methods across three key categories: (a) web-based method, (b) ephemeral method, (c) decentralized method
+2. Advance the above DID method standards to maturity in collaboration with appropriate Standards Development Organizations (SDOs) for each method (e.g., W3C for did:web / did:tdw)
+3. Develop approaches for scalable, independent DID methods standardization (such as grouping by traits), enabling organizations to drive and govern their own DID method standardization efforts
+4. Facilitating broader community coordination to support DID method standardization efforts across the ecosystem, including: 
+    1. Engaging with various standards groups, organizations, and individuals interested in DID method creation and standardization
+    2. Promoting knowledge sharing and best practices across the community 
+    3. Enabling discovery of efforts and collaborators
+
+## Objectives
+
+To achieve these goals, we will pursue the following objectives:
+
+### DIF Working Group
```

Comment:
```suggestion
### DIF Working Group with Joint Participation
```

I know it's happening under DIF, but can we further highlight that this is a joint initiative between multiple pre-SDOs?